CMakeLists.txt

```diff
@@ -118,6 +120,7 @@ add_subdirectory(${RADIUM_SRC_DIR})
 # ----------------------------------------------------------------------------------------------------------------------
 #   Applications
 #   Set Base directories for dependencies
+set( OMIT_MAIN_APP False CACHE BOOL "Choose to build or not the main radium application" )
```

Comment:
Seems like a divergence between two branches ... 
but I think it is good to keep it